playbook/app/pb_kits/playbook/pb_dialog/_dialog.jsx

```diff
@@ -55,7 +55,7 @@ const Dialog = (props: DialogProps) => {
   const dataProps = buildDataProps(data)
 
   const dialogClassNames = {
-    base: classnames('pb_dialog', buildCss('pb_dialog', size)),
+    base: classnames('pb_dialog', buildCss('pb_dialog', size), className ? buildCss('pb_dialog', className) : null),
```

Comment:
`className` needs to be last on this list or CSS modules will not work. Can you correct this?

Reply:
This is been resolved by using the `portalClassName` prop.